### PR TITLE
Process doc for AV members

### PR DESCRIPTION
**Pull request by patmbolger**
Much of this is very basic but new members sometimes need this detail.

### CODE REVIEW

File: docs/dev_process.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+Reason: Your **local repo** *master* branch needs to stay in sync with the **project repo** *master* branch.  This is because you want to have your feature branch (to be created below) to have the latest project code before you start adding code for the feature you’re working on.
+
+    git checkout master
+    git fetch upstream
+    git merge upstream/master
+
+You should not have merge conflicts in step 3, unless you’ve made changes to your local *master* branch directly (which you should not do).
+
+## Step 3: Create the feature branch in your local and github repos
+Reason: All of your development should occur in feature branches - ideally, one branch per Waffle ticket.  This keeps your local *master* branch clean (reflecting *upstream* master branch), allows you to abandon development approaches easily if required (e.g., simply delete the feature branch), and also allows you to maintain multiple work-in-process branches if you need to (e.g., work on a feature branch while also working own fixing a critical bug - each in its own branch).
```
Comment (FedericoEsparza) [inline docs/dev_process.md]: on = own

File: docs/dev_process.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        git push -u origin forgot-password#78
+
+## Step 4: Develop the Feature
+Develop the code for your feature (or chore/bug) as usual.  You can make interim commits to your local repo if this is a particularly large feature that takes a lot of time.
+
+When you have completed development, make your final commit to the feature branch in your local repo.
+
+## Step 5: Update local repo **master** branch
+Didn’t we just do this is step 2?  Yes, but we should do it again in cause any commits have occurred to the *master* branch in the project repo since you performed step 1.
+
```
Comment (FedericoEsparza) [inline docs/dev_process.md]: You might want to add the branch change command explicitly here: `git checkout master`

File: docs/dev_process.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+        git push -u origin forgot-password#78
+
+## Step 4: Develop the Feature
+Develop the code for your feature (or chore/bug) as usual.  You can make interim commits to your local repo if this is a particularly large feature that takes a lot of time.
+
+When you have completed development, make your final commit to the feature branch in your local repo.
+
+## Step 5: Update local repo **master** branch
+Didn’t we just do this is step 2?  Yes, but we should do it again in cause any commits have occurred to the *master* branch in the project repo since you performed step 1.
```
Comment (FedericoEsparza) [inline docs/dev_process.md]: case = cause

File: docs/dev_process.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+## Background
+
+This document outlines the process to be followed - for Agile Ventures members - in developing and
+submitting a pull request for this project.  The same process is used
+regardless of whether the pull request is for a feature, a bug or a chore.
+
+We use Waffle as our story board for this project.  The waffle board for this
+project is [here](https://waffle.io/WikiEducationFoundation/WikiEduDashboard).
+
+Make sure you are added as a "collaborator" to the project - confirm with the AV coordinator if there is any question.
```
Comment (hin101) [inline docs/dev_process.md]: Maybe we should add here how to be added as a collaborator.... i.e contact the slack channel

File: docs/README.md [hunk 5]
```diff
@@ -10,6 +10,7 @@
 
 ### Other
 - [Contributing](../CONTRIBUTING.md)
+- [Development Process (for AV Members)](dev_process.md)
```
Comment (ragesoss) [inline docs/README.md]: Let's not use the "AV" abbreviation in the main Readme.

**Pull request merged**